### DISCUSSION TITLE
Configure AdGuardHome for privacy purpose

### DISCUSSION THREAD

**Discussion by mmtech2012** [AdguardTeam/AdGuardHome#3164] [Q&A]
Hi Adguard Team
thank for your great app
how to config AdguardHome to pass DNS leak test?
for example:
https://whoer.net/dns-leak-test
https://browserleaks.com/dns
https://www.expressvpn.com/dns-leak-test
Thank you

**Comment by ainar-g** [comment]
As far as I know, the first step is to only use encrypted upstreams.  I think @ameshkov might add more tips.

**Reply by mmtech2012** [reply]
I already  used but nothing changed

**Reply by ameshkov** [reply]
Oh my, DNS leak tests. You are not supposed to pass a test like that, they're trying to sell you a VPN or a DNS and that's kinda it.

**Reply by mmtech2012** [reply]
you mean dns leak isn't  privacy issue at all?

**Reply by ameshkov** [reply]
Exactly
